jekyll/_cci2/orb-author-intro.md

```diff
@@ -8,88 +8,60 @@ order: 1
 ---
 ## Introduction to Orb Authoring
 
-Orbs are reusable packages of CircleCI configuration that you may share across projects, enabling you to create encapsulated, parameterized commands, jobs, and executors that can be [used across multiple projects]({{ site.baseurl }}/2.0/using-orbs/). 
+[Orbs]({{site.baseurl}}/2.0/orb-intro/) are [reusable packages of CircleCI configuration]({{site.baseurl}}/2.0/reusing-config/), enabling you to create parameterized commands, jobs, and executors that can be [used across multiple projects]({{site.baseurl}}/2.0/using-orbs/).
```

Comment:
I think we want to link to `orb-concepts` here, rather than `using-orbs` ... I think the content is identical but we decided to bin the whole "using orbs" section...